src/components/Notifications/TimeConfig.tsx

```diff
@@ -45,26 +61,27 @@ export const TimeConfigComponent: React.FunctionComponent = () => {
         setShowCustomSelect(true);
     }, []);
 
-    const handleTimeSelect = React.useCallback(() => {
+    const handleTimePrefSelect = React.useCallback((time) => {
+        setTimeSelect(time);
         setIsOpen(false);
-    }, []);
+    }, [ ]);
+
+    const handleButtonSave = React.useCallback(() => {
+        if (timeSelect) {
+            const mutate = saveTimePreference.mutate;
+            mutate({
+                body: timeSelect
+            }).then((response) => {
+                if (response.status === 200) {
```

Comment:
This is changing to 204 (No content) - I'll prepare a followup PR to sync after we merge https://github.com/RedHatInsights/notifications-backend/pull/2069